docs/docs/how-tos/nebari-aws.md

```diff
@@ -0,0 +1,132 @@
+export AWS_SECRET_ACCESS_KEY="Secret Access Key"
+```
+
+If you are using an already existing IAM user or want to use a different par of access keys, please refer to [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey) for detailed information on how to manage your IAM user's access keys.
+
+:::tip
+This variables applies only to your current shell session. If you want the variable to apply to future shell sessions, set the variable in your shell startup file, for example in the `~/.bashrc` or `~/.profile` file.
+:::
+
+:::note
+The following steps assumes you have completed the [Install Nebari](/started/installing-nebari) section, has confirmed that `nebari` is successfully installed in your environment and opted for **AWS** as your cloud provider. as well as has already configured the `nebari` environment variables. If you had any issues during the installation, please visit our getting started [troubleshooting](/started/troubleshooting) section for extra advice.
+:::
+
+
+## Nebari Initialize
+
+Great, you’ve gone through the [Nebari Installation](/started/installing-nebari.md) and [Authentication setup](#authentication) steps, and have ensured that all the necessary environment variables have been properly set. It is time to initialize and deploy Nebari!
+
+In your terminal, we advise you to start by creating a new project folder. Here, we will name the new folder as `nebari-aws`:
+
+```bash
+mkdir nebari-aws && cd nebari-aws
+```
+When you first initialize Nebari, you will be creating a `nebari-config.yaml` that contains a collection of preferences and settings for your deployment. The command bellow will generate a basic config file with an infrastructure based on **AWS**, named **projectname**, where the domain will be **domain** and the authetication mode set to **password**.
+
+```bash
+nebari init aws --project projectname --domain domain --auth-provider password
+```
+:::note Note
+You will be prompted to enter values for some of the choices above if they are omitted as command line arguments (for example project name and domain)
+:::
+
+Once `nebari init` is executed, you should then be able to see the following output:
+```bash
+Securely generated default random password=*** for Keycloak root user
+stored at path=/tmp/NEBARI_DEFAULT_PASSWORD
+INFO:botocore.credentials:Found credentials in environment variables.
+```
+:::tip
+MacOS generates a programmatic directory stored in `/private/var` and defines the `$TMPDIR` environment variable for locating the system temporary folder. We advise the user to look for the `NEBARI_DEFAULT_PASSWORD` file in the following  `/var/folders/xx/xxxxx/T` path.
+:::
+
+You can see that Nebari is generating a random password for the root user of Keycloak. This password is stored in a temporary file and will be used to authenticate to the Keycloak server once Nebari's infrastructure is fully deployed.
```

Comment:
What happens to this password? Why is it stored in a temporary file? Doesn't that run the risk of locking us out of our Nebari deployment? 

If the plan is to create one or more non-root administrator users as part of standing up the Nebari instance, it would be good to mention that here. Even if this Ubuntu-like superuser workflow is common in some circles, I still think it would be a good idea to be explicit here about how this will be handled.

Suggested change, assuming my understanding is correct:

```suggestion
You can see that Nebari is generating a random password for the root user of Keycloak. This password is stored in a temporary file and will be used to authenticate to the Keycloak server once Nebari's infrastructure is fully deployed, in order to create the first user accounts for administrator(s).
```

Reply:
> What happens to this password? Why is it stored in a temporary file? Doesn't that run the risk of locking us out of our Nebari deployment?

The same password is stored in the `nebari-config` file, so you can always use that even when the credential is missing in the env. 

> If the plan is to create one or more non-root administrator users as part of standing up the Nebari instance, it would be good to mention that here. Even if this Ubuntu-like superuser workflow is common in some circles, I still think it would be a good idea to be explicit here about how this will be handled.

The idea is to have only `root` as the administrator of keycloak, and we advise the user to change the password. But all this is out of the scope of installation and deployment, this point is addressed in the [Keycloak docs](https://docs.qhub.dev/en/stable/source/installation/login.html?#change-keycloak-root-password) of Nebari (I linked the current ones).